app/src/main/java/de/westnordost/streetcomplete/data/OsmModule.java

```diff
@@ -62,18 +65,18 @@ public static OsmConnection osmConnection(OAuthConsumer consumer)
 	}
 
 	@Provides public static OverpassMapDataDao overpassMapDataDao(
-			Provider<OverpassMapDataParser> parserProvider)
+			Provider<OverpassMapDataParser> parserProvider, SharedPreferences prefs)
 	{
 		OsmConnection overpassConnection = new OsmConnection(
-				OVERPASS_API_URL, ApplicationConstants.USER_AGENT, null);
+			prefs.getString(Prefs.OVERPASS_URL, "https://overpass-api.de/api/"), ApplicationConstants.USER_AGENT, null);
```

Comment:
I admit that after modifying this I was not expecting that everything will just work, as expected compilation failure.

So maybe I missed something else that should be done.

Reply:
```suggestion
			prefs.getString(Prefs.OVERPASS_URL, OVERPASS_API_URL), ApplicationConstants.USER_AGENT, null);
```

Same here? (I have no idea what I am suggesting, but well… :stuck_out_tongue_winking_eye: )